code/game/objects/structures/props.dm

```diff
@@ -1042,26 +1042,26 @@
 		helmet.forceMove(loc)
 		helmet = null
 	if(tagged)
-		var/obj/item/dogtag/new_info_tag = new(loc)
+		var/obj/item/information_tag/new_info_tag = new(loc)
 		new_info_tag.fallen_names = list(dogtag_name)
 		new_info_tag.fallen_assgns = list(dogtag_assign)
 		new_info_tag.fallen_blood_types = list(dogtag_blood)
 		fallen_list_cross -= dogtag_name
 	return ..()
 
 /obj/structure/prop/wooden_cross/attackby(obj/item/W, mob/living/user)
-	if(istype(W, /obj/item/dogtag))
-		var/obj/item/dogtag/dog = W
+	if(istype(W, /obj/item/information_tag))
+		var/obj/item/information_tag/tag = W
 		if(!tagged)
 			tagged = TRUE
 			user.visible_message(SPAN_NOTICE("[user] drapes the [W] around the [src]."))
```

Comment:
This doesn't use \the when it should, but regardless it should just be draping a single information tag. Currently it is this:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/8f478bab-242d-4a03-ba0b-fd1d0c41a83b)